### PR TITLE
Some rules also match any host, use trivial_cookie_name_c for those also

### PR DESCRIPTION
**Pull request by funkydude**

### CODE REVIEW

File: chromium/rules.js [hunk 1]
```diff
@@ -42,7 +42,11 @@ function Exclusion(pattern) {
  * @constructor
  */
 function CookieRule(host, cookiename) {
-  this.host_c = new RegExp(host);
+  if (host === ".*" || host === ".+" || host === ".") {
+    this.host_c = trivial_cookie_name_c;
```
Comment (semenko) [inline chromium/rules.js]: Could you please make a new `const` above for host rules, e.g. trivial_cookie_host?

**Comment by semenko** [comment]
Awesome -- thanks! LGTM other than the one comment above.

**Comment by funkydude** [comment]
If it's the name that bothers you, then why not rename it to e.g. `trivial_cookie_regex` rather than create another const of the same regex?

**Comment by semenko** [comment]
A single variable is cheap, so please make a new one, since it helps clarity & fits well with the parallelism for rules.

(Long term, it'll also make testing easier, since we can more cleanly test object equivalence.)

**Comment by funkydude** [comment]
Personally I think it's a waste and I'd go for renaming the var but oh well, changed.

**Comment by semenko** [comment]
Awesome - thanks :)

**Comment by jsha** [comment]
LGTM, thanks!

**Pull request merged**